tests/templates/kuttl/logging/01-install-hdfs-vector-aggregator.yaml

```diff
@@ -5,7 +5,7 @@ commands:
   - script: >-
       helm install hdfs-vector-aggregator vector
       --namespace $NAMESPACE
-      --version 0.39.0
+      --version 0.42.1
```

Comment:
Should this be 46.1?

Reply:
Sadly not. The chart version is now _so_ close to the app version.

I wanted to add a comment on the side, but that doesn't work in yaml with `>-`.